### PR TITLE
Chunking and Dask notebooks review

### PR DESCRIPTION
**Pull request by guillaumeeb**
Here are the corrections corresponding to the review I made on https://github.com/pangeo-data/foss4g-2022/pull/87.

### CODE REVIEW

File: tutorial/pangeo101/chunking_introduction.ipynb [hunk 1]
```diff
@@ -214,7 +214,7 @@
     "\n",
```
Comment (tinaok) [inline tutorial/pangeo101/chunking_introduction.ipynb:214]: <p>Please do not take out </p><p> </p><pre class="ql-syntax" spellcheck="false">Let's first re-open the same dataset without any chunking.
LTS=xr.open_dataset(fs.open(s3path))      
</pre><p> </p><p>This is to have 'simple' Dask graphe showing 8 chunks as described later.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-data/foss4g-2022/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (guillaumeeb) [inline tutorial/pangeo101/chunking_introduction.ipynb:214]: <p>I removed the re-openning here. I know Tina wanted it, but with the current version it's only using Numpy at first, and the quad chunks opening above works perfectly fine with all the comments afterwards.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-data/foss4g-2022/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='18'/>

**Comment by guillaumeeb** [comment]
<p>Crap, I really thought I did removed them. I had to go get my kids, I will look at your comments later!</p>

---

 <a href='https://app.reviewnb.com/pangeo-data/foss4g-2022/pull/93/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: tutorial/pangeo101/chunking_introduction.ipynb (hunk 1 above)
Comment (guillaumeeb) [reply tutorial/pangeo101/chunking_introduction.ipynb:214]: <p>I say below why I did that (but just after your review).</p><p> </p><p>The problem with this line is that it opens that dataset as a Numby data-array first. So I think we should at least put back the open_mfdataset call. But this is also a bit weird to reopen like that.</p><p> </p><p>Below we say </p><p> </p><blockquote>However, as you can see in the graph visualization above, Xarray will actually have to fetch at least one entire initial chunk that was defined when opening the Dataset at first before rechunking at a smaller size or even selecting one value</blockquote><p> </p><p>I think this is true by having only one chunk for the entire array, or having 4 chunks. In the end, I think having several chunks is more realistic too.</p><p> </p><p>All that is why I propose to stay with the same 4 chunks dataset.</p><p> </p><p> </p>

**Comment by guillaumeeb** [comment]
<p>I was under the impression that load() was often used (at least on Pangeo gallery), so I wanted to be complete about the method that can be used. But this is not a strong advice.</p>

---

 <a href='https://app.reviewnb.com/pangeo-data/foss4g-2022/pull/93/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by guillaumeeb** [comment]
<p>I'd stick with 'a Dask Distributed cluster', 'a Dask Distributed' alone sounds weird.</p><p> </p><p>And so I'd stick also with the description of the cluster first, and then with the tools allowing to deploy a Cluster, and connect to it.</p>

---

 <a href='https://app.reviewnb.com/pangeo-data/foss4g-2022/pull/93/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by guillaumeeb** [comment]
<p>Initially I just wanted to correct the form 'or ..' with ', etc.', and I just added another example of common (and some years ago popular) distributed computing infrastructure that can host Dask clusters.</p><p> </p><p>But I'm happy to remove Hadoop if you think it is too much.</p>

---

 <a href='https://app.reviewnb.com/pangeo-data/foss4g-2022/pull/93/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by guillaumeeb** [comment]
<p>Outputs cleared.</p>

---

 <a href='https://app.reviewnb.com/pangeo-data/foss4g-2022/pull/93/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by annefou** [comment]
Ok. What should we do with this PR? Shall we merge it? 

I will set up a new repo for the clover bootcamp and I am wondering if I should start from the existing repo or after merging this PR. Thanks!.

**Comment by guillaumeeb** [comment]
> What should we do with this PR? Shall we merge it?

I would say yes, but I'm a little biased 😄. Seriously, it needs some replies from @tinaok to see if we need to modify a few things.

**Pull request merged**